services/docker-compose-deploy.yml

```diff
@@ -30,6 +30,9 @@ services:
   migration:
     image: ${DOCKER_REGISTRY:-itisfoundation}/migration:${DOCKER_IMAGE_TAG:-latest}
 
+  osparc-gateway-server:
```

Comment:
so, is it now part of the stack? I thought that was not the use case?

Reply:
it is not part of the stack. but it is deployed to dockerhub